recipes/r-lasr/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "lasr" %}
+{% set version = "0.13.7" %}
+{% set posix = "m2-" if win else "" %}
+
+package:
+  name: r-{{ name }}
+  version: '{{ version }}'
+
+source:
+  url: https://r-lidar.r-universe.dev/src/contrib/lasR_{{ version }}.tar.gz
+  sha256: 746e84837af8a2d294321b76c790be5f0d57fc9e305a2e0f72703926a68f64b7
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - r-rcpp                       # [build_platform != target_platform]
+    - {{ compiler('c') }}          # [not win]
+    - {{ stdlib('c') }}            # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ stdlib('m2w64_c') }}      # [win]
+    - {{ compiler('cxx') }}        # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}make
+    - pkg-config
+    - posix                        # [win]
+    - llvm-openmp                  # [osx]
+  host:
+    - r-base
+    - r-rcpp
+    - r-r.utils
+    - geos
+    - libgdal-core
+    - libgomp  # [not osx]
+    - llvm-openmp  # [osx]
+    - proj
+  run:
+    - r-base
+    - llvm-openmp  # [osx]
+
+test:
+  commands:
+    - $R -e "library('lasR')"  # [not win]
+    - "\"%R%\" -e \"library('lasR')\""  # [win]
+
+about:
+  home: https://github.com/r-lidar/lasR
+  summary: Fast and pipeable airborne lidar processing tools. Read/write 'las' and 'laz' files, computation of metrics in area based approach, point filtering, normalization, individual tree segmentation and other manipulations in a powerful and versatile processing chain.
+  license: GPL-3.0-or-later
+  license_family: GPL3
+  license_file: LICENSE
```

This package does a bunch of vendoring. So those licenses need to also be explicitly captured:

```suggestion
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
    - LICENSE
    - LICENCE.note
    - inst/COPYRIGHTS
    - vendor/csf/LICENSE
    - vendor/delaunator/LICENSE
    - vendor/Eigen/COPYING.MINPACK
    - vendor/Eigen/COPYING.MPL2
    - vendor/Eigen/COPYING.README
    - vendor/geophoton/LICENSE
    - vendor/nlohmann/LICENSE
```

---

This is throwing and error on a build local test because the "vendor" paths don't actually exist. Pushing this change now though to see if conda-forge building handles it differently.

---

Ok it is a problem - so we need to generate the files as part of the recipe? Anyone have examples of this to follow https://conda-forge.org/docs/maintainer/adding_pkgs/#licenses-of-included-dependencies

---

@mfansler yup I eventually noticed that `src` was missing from the paths.